Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -547,7 +547,7 @@ export const getUTCAfterNSecondsImpl = function (str, seconds) {
 export const compareUTCDateImpl = function (date1, date2) {
   const d1 = new Date(date1);
   const d2 = new Date(date2);
-  return d1.getTime() - d2.getTime();
+  return Math.floor((d1.getTime() - d2.getTime())/1000); 
```

Comment:
Can we use `getDifferenceBetweenDates`  and remove this function

Reply:
`getDifferenceBetweenDates` is `ui-common` specific so I removed that function in `Utils.js` and used this only for both `ui-customer` and `ui-driver`.